app.js

```diff
@@ -0,0 +1,21 @@
+import { config } from 'dotenv';
+import nunjucks from 'nunjucks';
+import express from 'express';
+import meteorDelivery from './delivery/meteor-delivery.js';
+
+const app = express();
+const port = process.env.SERVER_PORT || 4000;
```

Comment:
add config.js file for managing all env variables

Reply:
Fixed.